README.md

```diff
@@ -35,19 +37,42 @@ The tutorial will create a Kubernetes Engine cluster that has a sample applicati
 
 ## Prerequisites
 
-The steps described in this document require the installation of several tools and the proper configuration of authentication to allow them to access your GCP resources.
+A Google Cloud account and project is required for this.  Access to an existing Google Cloud
+project with the Kubernetes Engine service enabled If you do not have a Google Cloud account
+please signup for a free trial [here](https://cloud.google.com).
+
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell](https://cloud.google.com/shell/docs/).
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fgke-monitoring-tutorial&page=editor&tutorial=README.md&image=gcr.io/graphite-cloud-shell-images/terraform)
+
+This Cloud Shell uses a custom container to install terraform, please select the 'Trust' radio button to
```

Comment:
Asking a user to trust the custom shell without also providing full transparency on what is in the container seems like a bad idea from a security/liability perspective since the container is then provided with the user's credentials. Do we have the source code to the image anywhere or has this image already been vetted in some way that I'm (very possibly) not aware of?

Reply:
We have an entitled security process to maintain the container. Also this is an in house Google container, maintained by another teams. I will send you the documentation.